contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -89,6 +88,26 @@ void multisig::approve( name proposer, name proposal_name, permission_level leve
             a.requested_approvals.erase( itr );
          });
    }
+
+   transaction_header trx_header = get_trx_header(prop.packed_transaction.data(), prop.packed_transaction.size());
+
+   if ( prop.earliest_exec_time.has_value() ) {
+      if ( prop.earliest_exec_time.value().has_value() ) {
+         return;
+      } else {
+         auto table_op = [](auto&&, auto&&){};
+         if ( approvals_satisfy_trx_authorization(proposer, proposal_name, prop.packed_transaction, table_op) ) {
+            auto prop_it = proptable.find( proposal_name.value );
+            proptable.modify( prop_it, proposer, [&]( auto& p ) {
+               p.earliest_exec_time = std::optional<time_point>{ current_time_point() + eosio::seconds(trx_header.delay_sec.value)};
+            });
+         } else {
+            return;
```

Comment:
This `return` is not necessary and I would prefer to see a style that doesn't return early in case we later want to add other code after this section that we expect to run.

Reply:
Previously linked: https://github.com/EOSIO/eosio.contracts/blob/deprecate-deferred3/contracts/eosio.msig/src/eosio.msig.cpp#L107-L116